docs/contracts/inheritance.rst

```diff
@@ -187,16 +188,20 @@ explicitly in the final override, but this function will bypass
         function destroy() public override(Base1, Base2) { super.destroy(); }
     }
 
-If ``Base2`` calls a function of ``super``, it does not simply
-call this function on one of its base contracts.  Rather, it
-calls this function on the next base contract in the final
-inheritance graph, so it will call ``Base1.destroy()`` (note that
-the final inheritance sequence is -- starting with the most
-derived contract: Final, Base2, Base1, Destructible, owned).
-The actual function that is called when using super is
-not known in the context of the class where it is used,
-although its type is known. This is similar for ordinary
-virtual method lookup.
+Calling a function of ``super`` still only calls a single function (the
+next one defined walking up the flattened inheritance list). However, if
+the coding convention of using ``super`` is also used by that function,
+the effect is that the calls "walk" up the inheritance list and call all
+definitions that are reachable that way. Thus, in the above example, the
```

Comment:
Honestly I find this sentence more confusing than before, especially the `call all definitions` part.